zanata-war/src/main/resources/db/changelogs/db.changelog-3.6.xml

```diff
@@ -30,4 +30,53 @@
       referencedTableName = "HProject" referencedColumnNames = "id"/>
   </changeSet>
 
+  <changeSet id="1" author="damason@redhat.com">
+    <comment>Create table HProject_LocaleAlias.</comment>
```

Comment:
It is mapping from class `HProject` to class `LocaleAlias`, so the `H` is just part of the class name that is represented by the id in the first column.